### PR TITLE
sf RNs: Sept 27

### PR DESCRIPTION
**Pull request by jshackell-sfdc**

### CODE REVIEW

File: releasenotes/README.md [hunk 1]
```diff
@@ -33,9 +33,17 @@ These changes are in the Salesforce CLI release candidate. We plan to include th
...
  
 * CHANGE: We removed the `package version retrieve` command that we announced on 9/15/23. This feature isn't quite ready for prime time, so we're removing it for now while we make improvements. We'll let you know after it's back up. (plugin-packaging PR [#447](https://github.com/salesforcecli/plugin-packaging/pull/447))
 
+* FIX: When you create a package version with the `package version create` command, the command now searches for profiles in only the package directories specified by the original `package create` command.   (GitHub issue [#2336](https://github.com/forcedotcom/cli/issues/2336), packaging PR [#397](https://github.com/forcedotcom/packaging/pull/397))
```
Comment (shetzel) [inline releasenotes/README.md]: It searches for profiles within all package directories defined within `sfdx-project.json`.

File: releasenotes/README.md [hunk 2]
```diff
@@ -33,9 +33,17 @@ These changes are in the Salesforce CLI release candidate. We plan to include th
...
+
+* NEW: Poll for status of a deployment when you run `project deploy report` with the new `--wait` flag. If you specify this flag, the command polls for the status every second until the timeout of `--wait` minutes.  If you don't specify the `--wait` flag, the command simply checks and displays the status of the deploy; the command doesn't poll for the status.
+
+    We also added the `--target-org` flag to `project deploy report`.  You usually don't need to specify this flag because the cached deploy job already references the org to which you deployed.  But if you run `project deploy report` on a computer different than the one from which you originally deployed, then you must specify the `--target-org` and it must point to the same org. If it doesn't, you get an error.
```
Comment (shetzel) [inline releasenotes/README.md]: Small nit-pick: If a default org is specified in the project for the other computer and it matches the org that was deployed to, then they won't have to set `--target-org`

**Review by shetzel — CHANGES_REQUESTED** [review]

File: releasenotes/README.md (hunk 1 above)
Comment (jshackell-sfdc) [reply releasenotes/README.md]: Thx!  Updated the RN

File: releasenotes/README.md (hunk 2 above)
Comment (jshackell-sfdc) [reply releasenotes/README.md]: Good catch -- I updated the text.

**Comment by jshackell-sfdc** [comment]
@shetzel , I inputted your feedback in case you want to take another look.  Thx!

**Pull request merged**